pwndbg/ida.py

```diff
@@ -97,19 +97,13 @@ def available():
 
 
 def l2r(addr):
-    exe = pwndbg.elf.exe()
-    if not exe:
-        raise Exception("Can't find EXE base")
-    result = (addr - int(exe.address) + base()) & pwndbg.arch.ptrmask
-    return result
+    vaddr = pwndbg.vmmap.find(pwndbg.regs.pc).vaddr
+    return (addr - vaddr + base()) & pwndbg.arch.ptrmask
 
 
 def r2l(addr):
```

Comment:
Since we are touching it, can we add docstring or rename  `r2l` and `l2r` so that the purpose of those functions would be more clear?
It rebases the addreses, isn't it? But which one rebases to gdb and which one to ida?